apps/explorer/src/components/orders/BridgeDetailsTable/index.tsx

```diff
@@ -0,0 +1,318 @@
+                    </a>
+                  ) : (
+                    bridgeDetails.providerName
+                  )}
+                </ProviderDisplayWrapper>
+              )}
+            </DetailRow>
+
+            {ownerAddress && bridgeDetails?.source && (
+              <DetailRow label="From" tooltipText={tooltipTextMap.ownerAddress} isLoading={isOverallLoading}>
+                <RowWithCopyButton
+                  textToCopy={ownerAddress}
+                  contentsToDisplay={
+                    <LinkWithPrefixNetwork
+                      to={getExplorerLink(bridgeDetails.source.chainId, ownerAddress, ExplorerDataType.ADDRESS)}
+                      target="_blank"
+                    >
+                      {ownerAddress} ↗
+                    </LinkWithPrefixNetwork>
+                  }
+                />
+              </DetailRow>
+            )}
+
+            {receiverAddress && bridgeDetails?.destination && (
+              <DetailRow label="To" tooltipText={tooltipTextMap.receiverAddress} isLoading={isOverallLoading}>
+                <RowWithCopyButton
+                  textToCopy={receiverAddress}
+                  contentsToDisplay={
+                    <LinkWithPrefixNetwork
+                      to={getExplorerLink(bridgeDetails.destination.chainId, receiverAddress, ExplorerDataType.ADDRESS)}
+                      target="_blank"
+                    >
+                      {receiverAddress} ↗
+                    </LinkWithPrefixNetwork>
+                  }
+                />
+              </DetailRow>
+            )}
+
+            <DetailRow label="Status" tooltipText={tooltipTextMap.status} isLoading={isOverallLoading}>
+              {bridgeDetails?.status ? <StatusLabel status={bridgeDetails.status} /> : null}
+            </DetailRow>
+
+            <DetailRow label="Amounts" tooltipText={tooltipTextMap.amounts} isLoading={isOverallLoading}>
+              {bridgeDetails && bridgeDetails.source && bridgeDetails.inputAmount ? (
+                <AmountSectionWrapper>
+                  <BridgeAmountDisplay
+                    labelPrefix="From:"
+                    bridgeToken={bridgeDetails.source}
+                    amount={bridgeDetails.inputAmount}
+                    isLoading={isOverallLoading}
+                  />
+                  <BridgeAmountDisplay
+                    labelPrefix={bridgeDetails.outputAmount ? 'To:' : 'To (est.):'}
+                    bridgeToken={bridgeDetails.destination}
+                    amount={bridgeDetails.outputAmount}
+                    isLoading={isOverallLoading}
+                  />
+                </AmountSectionWrapper>
+              ) : isOverallLoading ? null : (
+                'N/A'
+              )}
+            </DetailRow>
+
+            <DetailRow label="Costs & Fees" tooltipText={tooltipTextMap.costsAndFees} isLoading={isOverallLoading}>
+              {bridgeDetails &&
+                (() => {
```

Comment:
It's better to extract this function as a separate component because here it actually serves as a render function and we loose react component optimizations here.